keps/sig-apps/4026-crojob-scheduled-timestamp-annotation/README.md

```diff
@@ -107,16 +107,15 @@ to implement this enhancement.
 
 ##### Unit tests
 
-- `k8s.io/kubernetes/pkg/controller/cronjob`: `05/22/2023` - `96.2%`
+- `k8s.io/kubernetes/pkg/controller/cronjob`: `09/24/2023` - `71.2%`
 
 ##### Integration tests
 
-- Unit tests will ensure the new annotation is correctly added to jobs.
-- The integration test should ensure the annotation is present when the feature is on and missing when off. It will also verify that the annotation is only added to jobs from newly created CronJobs, not existing workloads.
+- No integration tests are planned for this feature.
 
 ##### e2e tests
 
-E2E tests will not provide any additional coverage that isn't already covered by unit + integration tests, since we are simply adding an annotation, so no e2e tests will be necessary for this change.
+- [CronJob should set the cronjob-scheduled-timestamp annotation](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/apps/cronjob.go#L264-282): [test coverage](https://storage.googleapis.com/k8s-triage/index.html?test=.*CronJob%20should%20set%20the%20cronjob-scheduled-timestamp%20annotation.*)
```

Comment:
This will be true once https://github.com/kubernetes/kubernetes/pull/127589 merges. I'll update it afterwards.